components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/dao/impl/PolicyDAOImpl.java

```diff
@@ -1343,4 +1343,32 @@ private int readHighestPriorityOfPolicies() throws PolicyManagerDAOException {
         return priority;
     }
 
+    @Override
+    public int getPolicyCount() throws PolicyManagerDAOException {
+
+        Connection conn;
+        PreparedStatement stmt = null;
+        ResultSet resultSet = null;
+        int policyCount = 0;
+        try {
+            conn = this.getConnection();
+            String query = "SELECT COUNT(DM_POLICY.ID) FROM DM_POLICY";
+            stmt = conn.prepareStatement(query);
+            resultSet = stmt.executeQuery();
+
+            while (resultSet.next()) {
+                policyCount = resultSet.getInt(0);
```

Comment:
Fixed in commit [1]
[1] https://github.com/wso2/carbon-device-mgt/commit/5a4d2e20a48a68602a13f11b53b9c56add055d64